feedback/filters.py

```diff
@@ -156,6 +169,76 @@ def filter(self, qs, value):
         return qs.filter_flags(*value)
 
 
+class ComboTextSearchWidget(forms.MultiWidget):
+    """Widget for displaying a text search field with a checkbox
+    that allows using another search method.
+    The checkbox label and helptext need to be defined through widget
+    attributes 'bool_label' and 'bool_helptext'.
+    """
+    template_name = "feedback/widgets/combo_textsearch_widget.html"
+
+    def __init__(self, attrs=None):
```

Comment:
`__init__` should have a return type annotated with `-> None`:
https://peps.python.org/pep-0484/#the-meaning-of-annotations